Source/Core/Core/PowerPC/Jit64/Jit_FloatingPoint.cpp

```diff
@@ -444,7 +444,7 @@ void Jit64::fmaddXX(UGeckoInstruction inst)
         if (round_input)
           Force25BitPrecision(XMM1, Rc, XMM2);
         else
-          MOVSD(XMM1, Rc);
+          MOVAPD(XMM1, Rc);
```

Comment:
Please add a comment we don't care about the upper half here, and the other places too.

Definitely better if `Rc` is a register, but if it is a memory location we will still be loading 128 bits. You may want to use `MOVSD` in that case, which surprisingly will set the upper 64 bits to 0. I don't think it matters to much, but I know we do this in a few places.

Reply:
Good point, I forgot that Rc could be a memory location here.

Personally, I think if I add an if statement so we use MOVSD only when it's a memory location, it's sufficiently clear that we don't care about the upper half that a comment isn't needed. Would you agree with this?

Reply:
Yeah, that works! 👍